src/main/app/src/components/Etusivu.tsx

```diff
@@ -68,8 +68,13 @@ export const Etusivu = () => {
   });
 
   const uutislinkit = uutiset?.['etusivun-uutiset']?.linkit ?? [];
+  const [showMore, setShowMore] = useState(true);
 
-  const [showMore, setShowMore] = useState(size(uutislinkit) <= 3);
```

Comment:
Jos nyt oikein hahmotan, tämä oli alunperin väärin päin ("näytä lisää" näytettiin vain kun uutislinkkejä on 3 tai alle), ja toimi koska ennen latauksen valmistumista linkkejä on 0.